5-Clustering/1-Visualize/translations/assignment.es.md

```diff
@@ -0,0 +1,11 @@
+# Investiga otras visualizaciones para agruparlas
+
+## Instrucciones
+
+En esta lección, has trabajado con algunas técnicas de visualización para así comprender cómo graficar tus datos en preparación para agruparlos. Los gráficos de dispersión son particularmente útiles para encontrar grupos de objetos. Investiga distintas formas y librerías para crear gráficos de dispersión y documenta tu trabajo en un notebook. Puedes usar los datos de esta lección, otras lecciones, o datos que tú mismo proporciones (por favor, dá crédito a su fuente en tu notebook). Grafica algunos datos usando gráficos de dispersión y explica lo que descubras.
```

Comment:
Suggestion:

remove the así from para así comprender

En esta lección, has trabajado con algunas técnicas de visualización para comprender cómo graficar tus datos en preparación para agruparlos.

Reply:
Thanks!